src/IKEA/Tradfri/Traits/ProvidesId.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace IKEA\Tradfri\Traits;
+
+/**
+ * Trait ProvidesId.
+ */
+trait ProvidesId
+{
+    /**
+     * @var int
+     */
+    protected $_id;
+
+    /**
+     * Get Id.
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->_id;
+    }
+
+    /**
+     * Set Id.
+     *
+     * @param int $id
+     *
+     * @return static
+     */
+    public function setId(int $id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/Fahl-Design/ikea-tradfri-php/pullRequest?prid=2789116)